static/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Allow: /$
```

Comment:
I'm pretty sure the `$` mark is only useful in the router regex, marking the end of the line.